thoth/storages/graph/janusgraph.py

```diff
@@ -364,6 +364,26 @@ def create_pypi_package_version(self, package_name: str, package_version: str, *
 
         return python_package, has_version, python_package_version
 
+    def unsolved_runtime_environments(self, package_name: str, package_version: str) -> list:
+        """Get Unsolved Runtime Environment which are not connected and attached to python package."""
+        query = self.g.V() \
+            .has('__label__', 'python_package_version') \
+            .has('package_name', package_name) \
+            .has('package_version', package_version) \
+            .inE() \
+            .has('__label__', 'solved') \
+            .outV() \
+            .dedup() \
+            .aggregate('inter_result') \
```

Comment:
When reading this, "inter_result" does not help me what it supposed to be. For better reading, I would suggest to rename it to something like "solvers_solved_python_package_version" or something like that.